frontend/.eslintignore

```diff
@@ -9,4 +9,3 @@ public/lib
 Godeps
 @types
 dynamic-demo-plugin
-integration-tests
```

Comment:
This change resulted in the second commit of lint fixes.  I kept those changes separate so as to not pollute the Protractor removal changes.